stage1/init/init.go

```diff
@@ -165,6 +165,10 @@ func installAssets() error {
 	if err != nil {
 		return err
 	}
+	systemdSysusersBin, err := common.LookupPath("systemd-sysusers", os.Getenv("PATH"))
```

Comment:
hmmm why do we even allow PATH?
Should this be with systemd-shutdown, systemd, systemd-journald below?